Algorithm/QCAlgorithm.Universe.cs

```diff
@@ -542,6 +546,22 @@ private void ConfigureUnderlyingSecurity(Security security)
             }
         }
 
+        /// <summary>
+        /// Checks the UniverseSettings.MinimumTimeInUniverse and if it's one day or more it updates it to be TimeSpan.Zero.
+        /// A warning message is also displayed to the user.
+        /// </summary>
+        private void CheckLiveCoarseUniverseSettings()
+        {
+            if (_liveMode && UniverseSettings.MinimumTimeInUniverse >= TimeSpan.FromDays(1))
+            {
+                Debug("Warning: Coarse universe selection in Live mode does not always happen at the same time. " +
+                      "The current setting of UniverseSettings.MinimumTimeInUniverse (one day or more) could cause unexpected universe members. " +
+                      "For this reason the setting has been updated to TimeSpan.Zero.");
+
+                UniverseSettings.MinimumTimeInUniverse = TimeSpan.Zero;
```

Comment:
So it's now _impossible_ to set this to be greater than 1 day? I imagine some subset of users depend on the min time setting? I suppose the other way for a user to enforce a min time would be to manually track it in a local subscriptions list to feed back into the coarse selector. Is there a way to default the value to zero for coarse but permit it to be overridable?